proto/topsql_agent.proto

```diff
@@ -55,8 +73,9 @@ message TopSQLSubRequest {}
 
 message TopSQLSubResponse {
     oneof resp_oneof {
-        CPUTimeRecord record = 1;
+        CPUTimeRecord record = 1; // DEPRECATED: use TopSQLRecord instead
```

Comment:
Maybe we can drop it, as there is no existing user.

Reply:
OK, I have deleted it and tested on TiDB and NGM.